examples/amp-web-push/publisher-files/helper-frame.js

```diff
@@ -0,0 +1,277 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+class AmpWebPushHelperFrame {
+  constructor(options) {
```

Comment:
Added JSDocs.

These files were originally minified by using the Typescript playground site (www.typescriptlang.org/play/) and then the UglifyJS online demo. The new gulp build process removes this manual copying.